dataactvalidator/config/sqlrules/c23_award_financial_1.sql

```diff
@@ -1,20 +1,25 @@
+WITH award_financial_c23_{0} AS
+    (SELECT piid,
+    	transaction_obligated_amou,
+    	submission_id
+    FROM award_financial
+    WHERE submission_id = {0})
 SELECT
 	NULL as row_number,
 	af.piid,
 	(SELECT COALESCE(SUM(sub_af.transaction_obligated_amou::numeric),0) AS transaction_sum
 		FROM award_financial as sub_af WHERE submission_id = {0} AND sub_af.piid = af.piid) AS transaction_obligated_amou_sum,
```

Comment:
I think this should be doing a SELECT from `award_financial_c23_{0}` not `award_financial`. There's also the `WHERE submission_id = {0}` here too.